### ISSUE TITLE
Wrong main rtl name shown - 2.0.0

### ISSUE THREAD

**Issue by melizaa** [fisharebest/webtrees#2344]
We should use the 2 _HEB name as the main name on RTL pages and not the Hebrew 2 _AKA name, when both names exist, in a similar way to the ltr pages where we use the 1 NAME, not the 2 _AKA name

**Comment by fisharebest** [comment]
We use the *first* name that matches the script.  ROMN/PHON/_XXX names are all considered equal.

This is in accordance with the GEDCOM principle that when there is more than one version of data, the prefered one is included first.

**Comment by melizaa** [comment]
webtrees has sorted the name parts. They are not in the sequence that I used when I filled the names.
I filled the 2 _HEB name before the Hebrew 2 _AKA and 2 _MARR names.

Could we create a utility that would split the 1 NAME (Latin1), 2 and 3 name parts (Latin2 or other non Hebrew names), 2 and 3 name parts (Hebrew3), 2 and 3 _HEB (Hebrew4) to
1 NAME (Latin1), 2 and 3 name parts (Latin2 and other non Hebrew names, like Thai names)
1 NAME (Hebrew4) with 2 GIVN, 2 SURN and 2 and 3 name parts (Hebrew3).

To do it manually will be a lot of work and will probably miss some names.
